opentech/static_src/src/sass/apply/components/_table.scss

```diff
@@ -0,0 +1,133 @@
+$table-breakpoint: 'small-tablet';
+
+table {
+    width: 100%;
+    background-color: $color--white;
+    border-collapse: collapse;
+    table-layout: fixed;
+
+    thead {
+        display: none;
+
+        @include media-query($table-breakpoint) {
+            display: table-header-group;
+        }
+
+        tr {
+            &:hover {
+                box-shadow: none;
+            }
+        }
+    }
+
+    tr {
+        border: 1px solid $color--light-mid-grey;
+        transition: box-shadow 0.15s ease;
+
+        @include media-query($table-breakpoint) {
+            &:hover {
+                box-shadow: 0 6px 35px -13px $color--black-50;
+            }
+        }
+
+        @include media-query($table-breakpoint) {
+            border-top: 0;
+            border-right: 0;
+            border-bottom: 2px solid $color--light-grey;
+            border-left: 0;
+        }
+
```

Comment:
Could the above two includes be combined into one?

Reply:
yup - good spot. Thanks!